addons/better-terrain/editor/Dock.gd

```diff
@@ -101,6 +104,13 @@ func _ready() -> void:
 	tile_view.paste_occurred.connect(_on_paste_occurred)
 	tile_view.change_zoom_level.connect(_on_change_zoom_level)
 	tile_view.terrain_updated.connect(_on_terrain_updated)
+	
+	symmetry_options.select(0)
+	symmetry_options.item_selected.connect(_on_symmetry_selected)
```

Comment:
This is already connected in the editor/scene, so it's not needed here.

Reply:
👍 
this was actually a consequence of `@tool` mode updating those properties in-place and then me subsequently pushing those changes.
same goes for `.visible = false`, unless I'm mistaken